tests/integration/smt_server/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.scc.suse.de/connect_sp2
+
+ADD files/smt_current.rpm /root/smt_current.rpm
```

Comment:
moving this after the next command could speed up things when rebuilding the image